++ dirname ./startserver.sh
+ BASEDIR=.
+ SCRIPTPATH=./../../rtun-routing
+ export PYTHONPATH=./../../torpy-rtun-fork/
+ PYTHONPATH=./../../torpy-rtun-fork/
+ python3 ./../../rtun-routing/rtun.py -v ERROR -g NeelTorRelay2 -f rendezvous.txt -t peer1peer2 -n default -i 5 -d 6
Exception in thread Thread-977:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/vboxuser/Git/imt4905/rtun-routing/lsr.py", line 573, in runDijkstra
    for n , w in args[0][current_vertex].items():
KeyError: 'P5'
Exception ignored in: <module 'threading' from '/usr/lib/python3.9/threading.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 1428, in _shutdown
    lock.acquire()
KeyboardInterrupt: 
